app/assets/javascripts/controllers/credentials/credentials_controller.js

```diff
@@ -44,17 +49,36 @@ ManageIQ.angular.app.controller('credentialsController', ['$http', '$scope', 'mi
     }
   };
 
+  $scope.changeStoredPrivateKey = function() {
+    $scope.bChangeStoredPrivateKey = true;
+    $scope.bCancelPrivateKeyChange = false;
+    $scope.emsCommonModel.ssh_keypair_password = '';
+  };
+
+  $scope.cancelPrivateKeyChange = function() {
+    if($scope.bChangeStoredPrivateKey) {
+      $scope.bCancelPrivateKeyChange = true;
+      $scope.bChangeStoredPrivateKey = false;
+      $scope.emsCommonModel.ssh_keypair_password = '';
```

Comment:
@mansam This is designed to be a very generic angular controller for managing credential fields across all forms.
It would not be a good idea to introduce model names from other controllers here (referring to `emsCommonModel`)